svc/FetchService.js

```diff
@@ -6,10 +6,67 @@
  */
 import {XH, HoistService} from '@xh/hoist/core';
 import {Exception} from '@xh/hoist/exception';
-import {castArray} from 'lodash';
+import {stringify} from 'qs';
 
 @HoistService()
 export class FetchService {
+
+    /**
+     * Send an HTTP request to a URL, and decode the response as JSON.
+     *
+     * @param {Object} opts - options to pass through to fetch, with some additions.
```

Comment:
Could we avoid re-documenting the shared parameters in each of the 3 xxxJson() methods?  How about saying the opts object contains "options for FetchService.fetch()" and providing a {link} to it that method?   The current setup seems like a maintenance issue -- lets put as much of the shared  documentation in FetchService.fetch().....

Reply:
Agreed.  I have edited the comments to reduce repetition.